modules/persistent-storage-csi-group-snapshots-create.adoc

```diff
@@ -0,0 +1,134 @@
+pvc-1       Bound     pvc-a42d81b8-e3df-11ed-b5ea-0242ac120002   1Gi        RWO           48S
+----
++
+This example uses two PVCs
+
+. Label the PVCs to belong to a snapshot group:
+.. Label PVC pvc-0 by running the following command:
++
+[source, terminal]
+----
+$ oc label pvc pvc-0 group=myGroup
+----
++
+.Example output
+[source, terminal]
+----
+persistentvolumeclaim/pvc-0 labeled
+----
+.. Label PVC pvc-1 by running the following command:
++
+[source, terminal]
+----
+$ oc label pvc pvc-1 group=myGroup
+----
++
+.Example output
+[source, terminal]
+----
+persistentvolumeclaim/pvc-1 labeled
+----
++
+In this example, you are labeling PVC "pvc-0" and "pvc-1" to belong to group "myGroup".
+
+. Create a `VolumeGroupSnapshot` object to specify your volume group snapshot:
+.. Create a `VolumeGroupSnapshot` object YAML file with the following example file:
++
+.Example VolumeGroupSnapshot YAML file
+[source, yaml]
+----
+apiVersion: groupsnapshot.storage.k8s.io/v1beta1
+kind: VolumeGroupSnapshot <1>
+metadata:
+  name: <volume-group-snapshot-name> <2>
+  namespace: <namespace> <3>
+spec:
+  volumeGroupSnapshotClassName: <volume-group-snapshot-class-name> <4>
+  source:
+    selector:
+      matchLabels:
+        group: myGroup <5>
+----
+<1> The `VolumeGroupSnapshot` object requests creation of a volume group snapshot for multiple PVCs.
+<2> Name of the volume group snapshot.
+<3> Namespace for the volume group snapshot.
+<4> The `VolumeGroupSnapshotClass` name. This object is created by the administrator and describes how volume group snapshots should be created.
+<5> The name of the label used to group the desired PVCs for the snapshot. In this example, it is "myGroup".
+
+.. Create the `VolumeGroupSnapshot` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <volume-group-snapshot-filename>.yaml
+----
+
+.Results
+Individual volume snapshots are created according to how many PVCs were specified as part of the volume group snapshot.
+
+These individual volume snapshots are named with the following format: <prefix>-<hash of VolumeGroupSnapshot UUID+PVC UUID+timestamp>
+
+A label with the volume group snapshot name is added to the individual volume snapshots:
+
+.Example individual volume snapshot
+[source, yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: snapshot1
+  labels:
+    volumeGroupSnapshotName: <volume-group-snapshot-name> <1>
+spec:
+  source:
+    persistentVolumeClaimName: vsc1
+status:
+  volumeGroupSnapshotName: volume-group-snapshot-name
+----
+<1> The name of the volume group snapshot. This was specified in step 3.
+
+In the preceding example, two individual volume snapshots are created as part of the volume group snapshot.
+
+[source, terminal]
+----
+snapshot-62abb5db7204ac6e4c1198629fec533f2a5d9d60ea1a25f594de0bf8866c7947-2023-04-26-2.20.4
+snapshot-2026811eb9f0787466171fe189c805a22cdb61a326235cd067dc3a1ac0104900-2023-04-26-2.20.4
```

Comment:
These names need to be changed to show its a hash currently its having a timestamp not the hash.

Reply:
@Madhu-1 Do you have an example that I can use?

Reply:
we have it in Kubernetes csi doc https://kubernetes-csi.github.io/docs/group-snapshot-restore-feature.html#test-volume-group-snapshot-feature

Reply:
Thanks! Done.